### ISSUE TITLE
Responsive Web Design

### ISSUE THREAD

**Issue by avalan4e57** [avalan4e57/beginners-github-search-manager#8]
We need to make the app responsive. 
I think the best way will be to do it with the help of bootstrap.

**Comment by yohgenius** [comment]
hello, may i take this issue?

**Comment by avalan4e57** [comment]
@yohgenius sure thing. Can you please describe in few words how are you going to work on this issue? You're welcome to ask any questions.

**Comment by yohgenius** [comment]
I think i will apply bootstrap for it. I don't know how to explain it in details though.
What i need is the screen resolutions to be responsive, what is the project made of, and then what should i choose in the "select scope" for my personal access token.

Thanks

**Comment by avalan4e57** [comment]
What do you think if we choose screen, tablet and phone resolutions? 
I don't think that I got you about "token" and "what is the project made of" questions. Can you please explain it more clear?

**Comment by yohgenius** [comment]
Ya, i mean phone right now usually around 5" or 5,5" and for tablet it's 7-10.1". So we need to choose some base for the screen.
What i mean about token is from this BGSM main screen, on "How to use" point 3,
and the "what is the project made of", is this made using php, reactjs, or any other sources used in this project.

**Comment by avalan4e57** [comment]
Ok, I see. And still I don't know how to choose for better. Suppose it could be good if we choose the lower border of each. If you have any ideas about this you're welcome to share. 
As for token you don't need to choose any options. Just type the name and then hit the 'Generate Token' button at the bottom of the page.
As for the sources they are open and you can browse the repository for details. Let me know if something isn't clear or you can't understand what tech is used here or there. For now I can say that I use express.js for server part, jQuery for front and pug/jade for HTML templates

**Comment by yohgenius** [comment]
Okay then, i will start to work on it now.. :)
I will return to ask if there is a problem...

**Comment by avalan4e57** [comment]
Cool! Good luck to you @yohgenius . I'm here to answer any questions and to help with what could be done from my part. Let's make this project great again :)

**Comment by yohgenius** [comment]
hello, i'm sorry it will take a bit longer, because i haven't seen a .pug file before haha...

**Comment by avalan4e57** [comment]
@yohgenius no problem. Take your time. Pug templates have a very straightforward structure though. It shouldn't make a big problem I hope.

**Comment by avalan4e57** [comment]
@yohgenius if you don't mind you can just make a totally new markup based on what you see when you use the app. If you do so I could transform that stuff into pug syntax. Just in case if you're stuck with the problem of understanding pug.

**Comment by yohgenius** [comment]
Hello, sorry for my absence..
I'm having no problem with pugjs as it is as you say easy to understand.
a few days before i was looking to make a better view for mobile version like collapsible sidebar where you didn't have to scroll down to check the 'working on' and 'PR-ed' projects.
If you think i was taking too long, i'm sorry and you could pass it to someone else..
But before that i will keep trying it :D
And if you have a reference on how to apply properties like this "data-toggle="collapse"" to pugjs, i would be glad to try it

**Comment by avalan4e57** [comment]
Hey @yohgenius . I'm not trying to make you hurry. Just wanted to give you a hand if you were stuck with the problem. 
As for applying properties you can check the sources and find something like  `button(class='btn btn-success btn-sm' onclick='prIssue()')` for example. I guess that should help you.

**Comment by yohgenius** [comment]
I'm sorry if i sound like being rushed... but i actually happy working on this.

i have tried the () to enclose the classing etc, and the property is applied as i see it in the developer tool's inspect element. but i don't know why the effect is not applied.

**Comment by avalan4e57** [comment]
Hmm, obviously the problem is not about using pug. There should be something else. Maybe you should try to play with that stuff using html markup and then translate it into pug if everything works ok. Actually that was the way I did it while working on this project :)

**Comment by yohgenius** [comment]
ah, i see... it does brighten my mind.. haha
i will try your suggestion. thanks for sharing and wait again.. :D

**Comment by yohgenius** [comment]
Hello, i'm back again... and sorry again for my absence.
This is my current work (i just found out how to apply those bootstrap thingy haha)
I hope you still accept it and give some inputs about the issue (and btw it's not done yet)...
:D 

![bgsm2](https://user-images.githubusercontent.com/20897249/32045954-d362949e-ba6b-11e7-89e0-3eb24da4373e.png)

![bgsm](https://user-images.githubusercontent.com/20897249/32045762-4a3dc30a-ba6b-11e7-8cb5-effecabb0ec8.png)

**Comment by avalan4e57** [comment]
Looks pretty nice! The same with correcting my grammar mistakes :) I guess that should be mobile view? Maybe it is better to make a collapse navigation for this kind? I'm also interested in how have you implemented search issue in context of responsive stuff.

**Comment by yohgenius** [comment]
Yes, this is the mobile view. I make this collapse navigation because i don't want people to scroll down too far to see their marked issues :)
And i'm glad that you liked it, but if you have another suggestions, i am very open about it. Now i will continue to experiment with the view to make it prettier.

this is my current progress.. my creativity is at its end haha...
![bgsm4](https://user-images.githubusercontent.com/20897249/32278153-4d953b1a-bf47-11e7-8357-80b4900ef42c.png)

**Comment by avalan4e57** [comment]
I mean that if mobile view we don't want that navigation in row but rather a small button which shows nav links on click and appears in a kind of dropdown.

**Comment by avalan4e57** [comment]
Hi, @yohgenius . As I said it could be nice to deal in some way with search in mobile view. [Here's an example](https://getbootstrap.com/docs/3.3/components/#navbar-forms) of how it could be done. What do you think about implementing this feature?

**Comment by yohgenius** [comment]
yep, i think it could b done.. what do we search here? the projects title? and what about the default page? still an empty page?
